muacrypt/bingpg.py

```diff
@@ -48,7 +48,8 @@ def __init__(self, ret, cmd, out, err, extrainfo=None):
 
     def __str__(self):
         lines = ["GPG Command '%s' retcode=%d" % (self.cmd, self.ret)]
-        for name, olines in [("stdout:", self.out), ("stderr:", self.err)]:
+        for name, olines in [("stdout:", self.out.__str__()),
```

Comment:
This keep the InvocationFailure printable even if the type of out is bytes rather than str.
No idea if it's the proper way to do it though. Just noticed it fixes the issue i was seeing.